server/service/integration_core_test.go

```diff
@@ -7963,7 +7963,7 @@ func (s *integrationTestSuite) TestGetHostSoftwareUpdatedAt() {
 		LabelUpdatedAt:  time.Now(),
 		PolicyUpdatedAt: time.Now(),
 		SeenTime:        time.Now(),
-		NodeKey:         ptr.String(t.Name() + "1"),
+		NodeKey:         ptr.String(strings.ReplaceAll(t.Name(), "/", "_") + "1"),
```

Comment:
What was the failure?

Reply:
It appears our router does not support `/`, even if escaped to `%2F`. This pattern was used in another test, so it appears we (or someone) is aware of that.

The fail was a 404 since it didn't find the right route.